source/module_hsolver/hsolver_pw_sdft.cpp

```diff
@@ -1,12 +1,12 @@
 #include "hsolver_pw_sdft.h"
 
-#include <algorithm>
-
 #include "module_base/global_function.h"
 #include "module_base/timer.h"
 #include "module_base/tool_title.h"
 #include "module_elecstate/module_charge/symmetry_rho.h"
 
+#include <algorithm>
```

Comment:
which function in <algorithm> has been used?

Reply:
std::max in line 119 is defined in header <algorithm>, and this header is moved automatically by precommit